blocks/library/gallery/block.js

```diff
@@ -167,6 +190,7 @@ class GalleryBlock extends Component {
 			return [
 				controls,
 				inspectorControls,
+				dropZone,
```

Comment:
The dropzone is a component that's dependent visually on its parent. And in the image block it's included inside the "placeholder" instead of using the parent element of this component (which is unknown really from a block author's point of view).

Should we move this inside the placeholder and the gallery container instead?

Reply:
Moved I added first as a test, and then as things were working great I kept it there, but you are totally right it is preferable to add it inside a container.